packages/mobile/src/components/details-tile/DetailsTile.tsx

```diff
@@ -347,6 +326,22 @@ export const DetailsTile = ({
         >
           {renderDetailLabels()}
         </View>
+        <View style={styles.descriptionContainer}>
```

Comment:
This is gonna change the layout of the Tracks/Playlist screens as well. I talked to Julian about this and he said he was ok with keeping the details labels consistent, so I think we should continue rendering them below the description

Reply:
Ah right, I was supposed to check in to see why this was backwards but we were in standup. Will update 👍